src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm.tsx

```diff
@@ -0,0 +1,661 @@
+                    variant="outlined"
+                    size="small"
+                    placeholder={t('Select Some Options')}
+                  />
+                )}
+              />
+            )}
+          </Box>
+
+          <Box mt={1} mb={1}>
+            <Typography display="inline">
+              {t('Add contacts with the following tag(s):')}
+            </Typography>
+            {!!contactTagsList.length && (
+              <Typography
+                display="inline"
+                className={classes.selectAll}
+                onClick={() => handleSelectAllTags(setFieldValue)}
+                data-testid="contactTagsSelect-selectAll"
+              >
+                {t('select all')}
+              </Typography>
+            )}
+
+            {loadingTags && <Skeleton height={40} />}
+
+            {contactTagsList && !loadingTags && (
+              <Autocomplete
+                multiple
+                autoHighlight
+                data-testid="contactTagsSelect"
+                id="tags-standard"
+                options={contactTagsList.filter(
+                  (tag1) => !tags.some((tag2) => tag2 === tag1),
+                )}
+                getOptionLabel={(option) => option}
+                value={tags}
+                onChange={(_event, values) => setFieldValue('tags', values)}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    variant="outlined"
+                    size="small"
+                    placeholder={t('Select Some Options')}
+                  />
+                )}
+              />
+            )}
+          </Box>
+          <Box mt={1} mb={1}>
+            <Typography>{t('Do not add contacts who:')}</Typography>
+            <Autocomplete
+              multiple
+              autoHighlight
+              data-testid="exclusionsSelect"
+              id="exclusions-standard"
+              options={contactExclusions.filter(
+                ({ value: id1 }) =>
+                  !exclusions.some(({ value: id2 }) => id2 === id1),
+              )}
+              getOptionLabel={(option) => option.name}
+              value={exclusions}
+              onChange={(_event, values) => setFieldValue('exclusions', values)}
+              renderInput={(params) => (
+                <TextField
+                  {...params}
+                  variant="outlined"
+                  size="small"
+                  placeholder={t('Select Some Options')}
+                />
+              )}
+            />
+          </Box>
+
+          {[errors.statuses, errors.tags, errors.exclusions].map(
+            (error, idx) => {
+              if (error && typeof error === 'string') {
+                return (
+                  <Alert severity="error" key={`error-${idx}`}>
+                    {error}
+                  </Alert>
+                );
+              }
+            },
+          )}
+
+          {!formRef && (
```

Comment:
So this ref lets the parent component submit the form? And if we don't have a ref, we add a submit button to submit the form? Do you mind adding a comment somewhere to that effect since that's a pattern devs may not be familiar with.

Reply:
Good point